data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,1660 @@
+			branch "know about the Pug"
+				has "main plot completed"
+			choice
+				`	"Speak to him about the weird aliens."`
+					goto warn
+				`	"Do not mention that to him."`
+					decline
+			label warn
+			action
+				log `I warned the Unfettered about the weird aliens north, they did not seem to care.`
+			`	You explain to the herald you have seen some weird aliens north, that could be allied to the Wanderers. Looking at the brief scans you took of them he seems to discard that warning as if it was nothing, "I do not know how those aliens have obtained jump drives, they most have stolen them somehow. Once we have taken Wanderer space they will prove to be a very handy supply of new jump drives..."`
+				decline
+			label "know about the Pug"
+			choice
+				`	"Speak to him about the Pug."`
+					goto warn
+				`	"Do not mention that to him."`
+					decline
+			`	You explain to the herald that you have seen weird aliens north, the Pug, which have had dealing with humanity before. He does not trust you when you speak of technology capable of breaking down hyperlanes, "there is no way an alien race with such primitive ships would have that kind of technology. They must have simply abused your primitive sensors. They must have stolen the jump drives, and I only see that as an opportunity for later." You try to convince him he should not take them lightly but he does not want to listen. "We have already done too much to prepare for war to turn back at the first sign of potential trouble."`
+			action
+				log `I warned the Unfettered about the Pug near the Wanderer, they did not seem to care.`
+
+
+
+mission "Unfettered: Cloaking Interest"
+	invisible
+	minor
+	source "Warfeed"
+	to offer
+		has "Unfettered: Special Cargo 2: done"
+		random < 20
+	on offer
+		require "Cloaking Device"
+		conversation
+			`A herald tells you He-Yi wants to see you again in his war room for a very generous proposition. He also mentions it's not urgent, so you can come back later if you're busy.`
+			choice
+				`	(Comply with the request.)`
+				`	(Later.)`
+					defer
+			`	You arrive to He-Yi standing proudly in front of a statue in his war room. "I had no idea you were a disciple of Tejolis as well."`
+			choice
+				`	"What do you mean?"`
+				`	(Let him continue speaking.)`
+			`	"I have discovered that you have somehow acquired an item I myself thought only a thing of legends. Something so [precious, rare] and yet so evasive. I want it. I am of course referring to the 'cloaking device.' I consider this item of the highest [importance, priority], and I am willing to give you 50 million for it. I will not lie to you: this item nearly [guarantees, ensures] the survival of the one who has it if he knows how to use it properly. It would also allow new strategies in our war against the bird people."`
```

Comment:
```suggestion
			`	"I have discovered that you have somehow acquired an item I myself thought only a thing of legends. Something so [precious, rare] and yet so evasive. I want it. I am of course referring to the 'cloaking device.' I consider this item of the highest [importance, priority], and I am willing to give you 50 million credits for it. I will not lie to you: this item nearly [guarantees, ensures] the survival of the one who has it if he knows how to use it properly. It would also allow new strategies in our war against the bird people."`
```